pkg/serverconfig/config.go

```diff
@@ -253,6 +253,14 @@ func addCustomization(fs *flag.FlagSet, customization *Customization) {
 			klog.Fatalf("Could not marshal ConsoleConfig customization.developerCatalog.categories field: %v", err)
 		}
 	}
+
+	if customization.ProjectAccess.AvailableClusterRoles != nil {
+		if projectAccess, err := json.Marshal(customization.ProjectAccess); err != nil {
+			klog.Fatalf("Could not marshal ConsoleConfig customization.projectAccess field: %v", err)
```

Comment:
Is this needed if we're already validating the flag set in validate.go?

Reply:
Makes sense. Removed the validation check.